examples/integrations/oci/atp-cdi/src/main/java/io/helidon/examples/integrations/oci/atp/cdi/ATPResource.java

```diff
@@ -0,0 +1,90 @@
+/*
+ * Copyright (c) 2021 Oracle and/or its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.helidon.examples.integrations.oci.atp.cdi;
+
+import java.io.InputStream;
+import java.nio.channels.Channels;
+import java.util.Optional;
+
+import javax.inject.Inject;
+import javax.ws.rs.DELETE;
+import javax.ws.rs.DefaultValue;
+import javax.ws.rs.GET;
+import javax.ws.rs.HeaderParam;
+import javax.ws.rs.POST;
+import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.StreamingOutput;
+
+import io.helidon.common.http.Http;
+import io.helidon.integrations.common.rest.ApiOptionalResponse;
+import io.helidon.integrations.oci.atp.OciAutonomousDB;
+import io.helidon.integrations.oci.atp.GenerateAutonomousDatabaseWallet;
+import io.helidon.integrations.oci.atp.GenerateAutonomousDatabaseWalletRx;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+/**
+ * JAX-RS resource - REST API for the atp example.
+ */
+@Path("/atp")
+public class ATPResource {
```

Comment:
Naming in Helidon is strictly "camel case" - so this class should be named `AtpResource` (this is valid for any class, field, method or variable)

Reply:
Updated all names to "camel case".